core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -79,6 +79,8 @@
   REQUEST_LOGGER_STACK_TRACES("request.tracker.logs.show-stack-traces", false),
 
   CONTROL_CONNECTION_TIMEOUT("connection.control-connection.timeout", true),
+  CONTROL_CONNECTION_ALLOW_PORT_DISCOVERY(
+      "connection.control-connection.allow-port-discovery", false),
```

Comment:
re separate tickets: it makes it easier to track progress across the two codebases. Otherwise we mark the JIRA ticket as "fixed" when it's merged to driver 3 but not yet to driver 4, or vice-versa.